[2023-11-29T21:59:51.005+0200] {taskinstance.py:1149} INFO - Dependencies not met for <TaskInstance: lviv_weather.extract_data manual__2023-11-29T19:58:52.282171+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2023-11-29T21:59:51.005+0200] {local_task_job_runner.py:154} INFO - Task is not able to be run
